src/jvmMain/kotlin/io/beatmaps/api/review.kt

```diff
@@ -386,4 +438,180 @@ fun Route.reviewRoute() {
             }
         }
     }
+
+    post<ReplyApi.Create> { req ->
+        requireAuthorization { _, user ->
+            if (user.suspended) return@requireAuthorization call.respond(ActionResponse(false, listOf("Suspended account")))
+
+            val reply = call.receive<ReplyRequest>()
+
+            captchaIfPresent(reply.captcha) {
```

Comment:
captchaIfPresent treats null as a valid captcha. The other endpoints that use it only allow updates when the captcha is null but there are no checks here I can see?